src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { Tab } from '../../types';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string,
+  onTabSelected: (id: Tab) => void
+};
+
+export const Tabs: React.FC<Props> = (
+  { tabs, selectedTabId, onTabSelected }: Props,
+) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
useMemo

Reply:
Yep, need to use useMemo hook. Array search is a complex operation.